### DISCUSSION TITLE
Question:Is it possible to verify the code running in enclave in simulation mode

### DISCUSSION THREAD

**Discussion by vishal-kanna** [edgelesssys/ego#296] [General]
How to verify that the code is running inside the enclave in simulation mode

**Comment by thomasten** [comment]
Hi, do you mean whether attestation can be done in simulation mode? Unfortunately no, the attestation process is not simulated and the APIs will return an error in simulation mode.

**Reply by vishal-kanna** [reply]
Ok @thomasten
